libraries/browser-tracker-core/src/tracker/types.ts

```diff
@@ -1,3 +1,4 @@
+import { ParsedIdCookie } from './id_cookie';
```

Comment:
Should we move the `ParsedIdCookie` type here to avoid the circular import of types between `types.ts` and `id_cookie.ts`?

Reply:
Good catch! Also migrated it to a labelled tuple which is hopefully a bit easier to work with.